test/e2e/harness/src/Home.js

```diff
@@ -60,7 +59,7 @@ export default withOktaAuth(class Home extends Component {
       <button id="logout-button" onClick={this.logout}>Logout</button> :
       <button id="login-button" onClick={this.login}>Login</button>;
 
-    const pkce = this.props.authService._oktaAuth.options.pkce;
+    const pkce = this.props.oktaAuth.options.pkce;
```

Comment:
Let's use the `isPKCE` method: https://github.com/okta/okta-auth-js/blob/master/lib/browser/browser.ts#L496